packages/main/src/TimePicker.js

```diff
@@ -0,0 +1,579 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
+import { fetchI18nBundle, getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import { getLocale } from "@ui5/webcomponents-base/dist/LocaleProvider.js";
+import ValueState from "@ui5/webcomponents-base/dist/types/ValueState.js";
+import DateFormat from "@ui5/webcomponents-utils/dist/sap/ui/core/format/DateFormat.js";
+import { fetchCldr } from "@ui5/webcomponents-base/dist/asset-registries/LocaleData.js";
+import PopoverHorizontalAlign from "./types/PopoverHorizontalAlign.js";
+import ResponsivePopover from "./ResponsivePopover.js";
+import PopoverPlacementType from "./types/PopoverPlacementType.js";
+import TimePickerTemplate from "./generated/templates/TimePickerTemplate.lit.js";
+import {
+	TIMEPICKER_HOURS_LABEL,
+	TIMEPICKER_MINUTES_LABEL,
+	TIMEPICKER_SECONDS_LABEL,
+	TIMEPICKER_PERIODS_LABEL,
+} from "./generated/i18n/i18n-defaults.js";
+import {
+	isLeft,
+	isRight,
+	isTabNext,
+	isTabPrevious,
+} from "../../base/src/events/PseudoEvents.js";
+
+// Styles
+import TimePickerCss from "./generated/themes/TimePicker.css.js";
+import ResponsivePopoverCommonCss from "./generated/themes/ResponsivePopoverCommon.css.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-timepicker",
+	properties: /** @lends sap.ui.webcomponents.main.TimePicker.prototype */ {
+		/**
+		 * Defines a formatted time value.
+		 *
+		 * @type {string}
+		 * @defaultvalue ""
+		 * @public
+		 */
+		value: {
+			type: String,
+			defaultValue: "",
+		},
+
+		/**
+		 * Determines the format, displayed in the input field.
```

Comment:
The format patterns are always tricky, write some examples, they are very useful.

/**
* Determines the format, displayed in the input field.
* <br>
* EXample

Reply:
Done